SalesforceCommerceCloud/pwa-kit
# Password change bug fix

# Description
Currently, after changing their password, customers cannot add products to their cart. 

To reproduce the problem:

1. Open https://pwa-kit.mobify-storefront.com/ and login as a registered shopper
2. Change your password in your Account Details page
3. Try to add a product to your basket -> Error

This fix involves restarting the "SLAS login dance" after a user changes their credentials (otherwise, both the SLAS access & refresh tokens will be invalid)



# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Pass customer email when calling `updatePassword`
- Re-login the customer with new credentials (email & updated pw)

# How to Test-Drive This PR

- Checkout this branch
- Run `npm start`
- Go to http://localhost:3000/
- Sign in as a registered user
- Go to "Account Details" page by clicking on Account icon
- Change password
- Navigate to a product and click "Add to Cart"
- Verify that product was successfully added!🥳

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
